docs/installation/windows.md

```diff
@@ -346,7 +346,7 @@ uses. You can do this with
 [puttygen](http://www.chiark.greenend.org.uk/~sgtatham/putty/download.html):
 
 1. Open `puttygen.exe` and load ("File"->"Load" menu) the private key from
-  `%USERPROFILE%\.ssh\id_boot2docker`
+  `%USERPROFILE%\.docker\machine\machines\<name_of_your_machine>`
```

Comment:
You could try changing to (make sure there are 8 spaces before `%USERPROFILE%`);

```Markdown
1. Open `puttygen.exe` and load ("File"->"Load" menu) the private key from this directory:

        %USERPROFILE%\.docker\machine\machines\<name_of_your_machine>

```